<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetup</class>
 <widget class="QDialog" name="DialogSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>443</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>KchmViewer settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Widget8">
      <attribute name="title">
       <string>General behavior</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="buttonGroup1">
           <property name="title">
            <string>If no arguments given on load</string>
           </property>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>9</number>
            </property>
            <item>
             <widget class="QRadioButton" name="m_radioOnBeginOpenDialog">
              <property name="text">
               <string>Open the 'open CHM file' dialog</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_radioOnBeginOpenLast">
              <property name="text">
               <string>Open the &amp;last used CHM file, if exist</string>
              </property>
              <property name="shortcut">
               <string>Alt+L</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="buttonGroup2">
           <property name="title">
            <string>History</string>
           </property>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>9</number>
            </property>
            <item>
             <layout class="QHBoxLayout">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="textLabel1">
                <property name="text">
                 <string>Store history information for</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="m_historySize">
                <property name="prefix">
                 <string/>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="textLabel2">
                <property name="text">
                 <string>latest files</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="m_rememberHistoryInfo">
              <property name="toolTip">
               <string>Remembers the information about the files, which were opened. Remembers last opened page, search history, font size, bookmarks and so on.</string>
              </property>
              <property name="text">
               <string>R&amp;emember extra information for history files</string>
              </property>
              <property name="shortcut">
               <string>Alt+E</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox1">
         <property name="title">
          <string>CHM security settings</string>
         </property>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>9</number>
          </property>
          <item>
           <widget class="QGroupBox" name="buttonGroup4">
            <property name="title">
             <string>When external link clicked</string>
            </property>
            <layout class="QVBoxLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="margin">
              <number>9</number>
             </property>
             <item>
              <widget class="QRadioButton" name="m_radioExtLinkOpenAlways">
               <property name="text">
                <string>Alwa&amp;ys open it in external browser</string>
               </property>
               <property name="shortcut">
                <string>Alt+Y</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="m_radioExtLinkAsk">
               <property name="text">
                <string>Ask for confirmation</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="m_radioExtLinkOpenNever">
               <property name="text">
                <string>Do not open the link</string>
               </property>
               <property name="shortcut">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="buttonGroup5">
            <property name="title">
             <string>When link pointed to a new CHM file clicked</string>
            </property>
            <layout class="QVBoxLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="margin">
              <number>9</number>
             </property>
             <item>
              <widget class="QRadioButton" name="m_radioNewChmOpenAlways">
               <property name="text">
                <string>Always open a new CHM file</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="m_radioNewChmAsk">
               <property name="text">
                <string>Ask for confirmation</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="m_radioNewChmOpenNever">
               <property name="text">
                <string>Do not open a new CHM file</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Widget9">
      <attribute name="title">
       <string>Version-specific settings</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="m_groupKDEsettings">
         <property name="title">
          <string>KDE version settings</string>
         </property>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>9</number>
          </property>
          <item>
           <widget class="QGroupBox" name="buttonGroup5_2">
            <property name="title">
             <string>To show HTML content, use</string>
            </property>
            <layout class="QVBoxLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="margin">
              <number>9</number>
             </property>
             <item>
              <widget class="QRadioButton" name="m_radioUseQtextBrowser">
               <property name="text">
                <string>&amp;QTextBrowser-based widget</string>
               </property>
               <property name="shortcut">
                <string>Alt+Q</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="m_radioUseKHTMLPart">
               <property name="text">
                <string>&amp;KHTMLPart-based widget</string>
               </property>
               <property name="shortcut">
                <string>Alt+K</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="m_radioUseQtWebkit">
               <property name="text">
                <string>Qt &amp;Webkit (requires Qt 4.4 or higher)</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupKhtmlSecurity">
            <property name="title">
             <string>KHTMLPart security</string>
            </property>
            <layout class="QGridLayout">
             <property name="margin">
              <number>9</number>
             </property>
             <property name="spacing">
              <number>6</number>
             </property>
             <item row="1" column="1">
              <widget class="QCheckBox" name="m_enableRefresh">
               <property name="text">
                <string>Enable &amp;refresh</string>
               </property>
               <property name="shortcut">
                <string>Alt+R</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="m_enableJava">
               <property name="text">
                <string>Enable &amp;Java</string>
               </property>
               <property name="shortcut">
                <string>Alt+J</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="m_enablePlugins">
               <property name="text">
                <string>Enable &amp;plugins</string>
               </property>
               <property name="shortcut">
                <string>Alt+P</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="m_enableJS">
               <property name="text">
                <string>Enable Java&amp;Script</string>
               </property>
               <property name="shortcut">
                <string>Alt+S</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QGroupBox" name="buttonGroup7">
         <property name="title">
          <string>View HTML source command uses</string>
         </property>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>9</number>
          </property>
          <item>
           <widget class="QRadioButton" name="m_advViewSourceInternal">
            <property name="text">
             <string>Built-in HTML source viewer</string>
            </property>
            <property name="shortcut">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="m_advViewSourceExternal">
            <property name="text">
             <string>E&amp;xternal program, specified below</string>
            </property>
            <property name="shortcut">
             <string>Alt+X</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="textLabel1_3">
              <property name="text">
               <string>External program:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_advExternalProgramName"/>
            </item>
            <item>
             <widget class="QPushButton" name="btnBrowse">
              <property name="text">
               <string>&amp;Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Other</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QCheckBox" name="boxLayoutDirectionRL">
            <property name="text">
             <string>Use right-to-left layout direction (Arabic adn Hebrew layouts)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="boxAutodetectEncoding">
            <property name="text">
             <string>Automatically detect encoding for KHTMLPart and WebKit browsers (not recommended)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>567</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>m_radioOnBeginOpenDialog</tabstop>
  <tabstop>m_radioOnBeginOpenLast</tabstop>
  <tabstop>m_historySize</tabstop>
  <tabstop>m_rememberHistoryInfo</tabstop>
  <tabstop>m_radioExtLinkOpenAlways</tabstop>
  <tabstop>m_radioExtLinkAsk</tabstop>
  <tabstop>m_radioExtLinkOpenNever</tabstop>
  <tabstop>m_radioNewChmOpenAlways</tabstop>
  <tabstop>m_radioNewChmAsk</tabstop>
  <tabstop>m_radioNewChmOpenNever</tabstop>
  <tabstop>m_radioUseQtextBrowser</tabstop>
  <tabstop>m_radioUseKHTMLPart</tabstop>
  <tabstop>m_enableJS</tabstop>
  <tabstop>m_enableJava</tabstop>
  <tabstop>m_enablePlugins</tabstop>
  <tabstop>m_enableRefresh</tabstop>
  <tabstop>m_advViewSourceInternal</tabstop>
  <tabstop>m_advViewSourceExternal</tabstop>
  <tabstop>m_advExternalProgramName</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_radioUseKHTMLPart</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupKhtmlSecurity</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_advViewSourceExternal</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_advExternalProgramName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_radioUseQtextBrowser</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupKhtmlSecurity</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSetup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
